src/main/scala/io/iohk/ethereum/db/dataSource/IodbDataSource.scala

```diff
@@ -30,7 +30,7 @@ class IodbDataSource private (lSMStore: LSMStore, keySize: Int, path: String) ex
 
   override def clear: DataSource = {
     destroy()
-    IodbDataSource(path, keySize, recreate = true)
```

Comment:
@LukasGasior1 yes. After discussing, `recreate` because It's not actually being used (at least what I've seen) and adds more complexity (that obviously needs to be tested). That being said, I thought it was better move to remove it until we need it. If that happens we can either look for the change in the git logs or maybe look for a better alternative to it.